tree/30_generic_methods/augeas_commands.cf

```diff
@@ -0,0 +1,79 @@
+#####################################################################################
+# Copyright 2020 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+#
+# @name Augeas Commands
+# @description Use augeas binaries to execute augtool command(s) directly on the agent. 
+# The generic method will return the command(s) execution output otherwise if the augeas isn't installed on the agent, it will produces an error.
+#
+# @parameter variable_prefix The prefix of the variable name
+# @parameter variable_name   The variable to define, the full name will be variable_prefix.variable_name
+# @parameter commands        The augeas command(s) 
+# @parameter noautoload      Activate the '--no autoload' option if you don't want augeas to charge all the files/lens, it's 'false' by default.
+# @parameter_constraint noautoload "allow_empty_string" : true
```

Comment:
I'd prefer to name the parameter "autoload" with a default to `true`. It is easier to handle "positive" boolean variables name (i.e. `notSomething = true` is more confusing than `Something = false`).

Reply:
okay changing this